core/control-plane/transfer-core/src/main/java/org/eclipse/edc/connector/transfer/command/handlers/NotifyStartedTransferCommandHandler.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.transfer.command.handlers;
+
+import org.eclipse.edc.connector.transfer.spi.observe.TransferProcessObservable;
+import org.eclipse.edc.connector.transfer.spi.store.TransferProcessStore;
+import org.eclipse.edc.connector.transfer.spi.types.TransferProcess;
+import org.eclipse.edc.connector.transfer.spi.types.command.NotifyStartedTransferCommand;
+
+import static org.eclipse.edc.connector.transfer.spi.types.TransferProcess.Type.CONSUMER;
+
+/**
+ * Puts a TransferProcess in the STARTED state as the counter-party actually started the transfer.
```

Comment:
small nit: To me the class name sounds a bit misleading: `NotifyStarted...` implies, that "I am notifying" someone. While, what actually happens, is that "I am notified, that something has happened". How about naming the class `CounterPartyStarted...` or `TransferHasStarted...`?

Reply:
@paullatzelsperger the idea was to have the command conjugated in the present tense, like a direction coming from the outside that tells something that must happen but it's not yet happened, yes, the provider will emit the `TransferStartedEvent` and that will cause on the provider the creation of a command that would be end in an action, in this case I think it could be treated as a notification because the only thing the consumer can do is to accept it and change the status.
Putting it in the past tense would create confusion between commands and events.